higher_health/templates/healthcheck_questionnaire.html

```diff
@@ -175,6 +175,11 @@ <h2 class="subheading">Confirmation</h2>
       {% endfor %}
     };
 
+    let other_varsity = '90';
```

Comment:
Does this have to be a numeric value? would `'other'` cause problems as a value? Just asking

Reply:
@miltontony yes, `'other'` would cause issues the `90` there would be the index ID for the "Other" Tertiary institution

Reply:
cool

Reply:
Please see Rudi's later comment though - it's better to not add this field to our DB at all and only use it as a UI value.